client/me/account/index.jsx

```diff
@@ -210,6 +210,13 @@ module.exports = React.createClass( {
 		} );
 	},
 
+	onSiteSelect( siteSlug ) {
+		let selectedSite = sites.getSite( siteSlug );
+		if ( sites.getSite( siteSlug ) ) {
```

Comment:
heh... you're right :smile: I created the variable so I didn't have to get the site twice, and then I did it anyways :panda_face: